build.sh

```diff
@@ -1134,8 +1139,11 @@ EOM
     "package-ios-swift")
         cd tightdb_objc
         for version in 8.0 8.1 8.2; do
-            REALM_XCODE_VERSION="$version" sh build.sh prelaunch-simulator
-            REALM_XCODE_VERSION="$version" sh build.sh ios-swift
+            REALM_XCODE_VERSION=$version
+            REALM_SWIFT_VERSION=
+            set_xcode_and_swift_versions
```

Comment:
`build.sh` now calls `set_xcode_and_swift_versions` unconditionally, so it's superfluous to call it in all these `package-` commands. Also, `REALM_SWIFT_VERSION=` could be moved outside these for loops.

```diff
-for version in 8.0 8.1 8.2; do
-REALM_XCODE_VERSION=$version
-REALM_SWIFT_VERSION=
-set_xcode_and_swift_versions
+REALM_SWIFT_VERSION=
+for REALM_XCODE_VERSION in 8.0 8.1 8.2; do
```